openstack/elektra/charts/postgres/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.backup.enabled }}
```

Comment:
I don't understand the purpose of this configmap? I don't see where it is being used.